specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/storagecache.json

```diff
@@ -2333,23 +2333,21 @@
           "description": "Properties of the amlFilesystem.",
           "properties": {
             "storageCapacityTiB": {
-              "description": "The size of this filesystem, in TiB.  This may be rounded up.",
+              "description": "The size of this filesystem, in TiB.  This may be rounded up. This field can't be supplied in modify operations.",
```

Comment:
must not be supplied, or must not be changed? Given the way the REST APIs work, it's pretty normal to do a GET, modify some fields in the result, then do a PUT. In many cases, APIs that do not allow a parameter to be modified after create still require it to be supplied when updating. This has made developers accustomed to this RMW cycle. For that reason, I prefer the semantic that it may be provided, but attempting to change it generates an error. Something like "This field may not be modified after creation."

Reply:
(comment applies throughout)

Reply:
This is actually a requirement for Microsoft APIs: PUTs must be idempotent, which means you must be allowed to supply attributes with their current values.    We don't need to allow changes, of course.

Reply:
Yeah- I think what I'm asking for is a rewording of the doc string, not a fundamental change. :-)

Reply:
Updated.